CHANGELOG.md

```diff
@@ -1,16 +1,223 @@
+
+  #### DOCUMENTATION CHANGES
+
+  * Correct the **Greek** language 2-char code from `gr` to `el` in the Semantic Versioning statement.
+  * Change references to **Mapzen** (RIP) to **Tilezen**.
+  * **Update MapboxGL demo**, thanks to Apollo Mapping
+  * Use service wording changes (Less > Fewer)
+  * Updated Layers documentation for v1.5 schema changes.
+  * TODO: Updated tilejson/tilejson.json.erb for v1.5 schema changes.
+  
+
+  #### INTERNAL CHANGES
+
+  * Change references to **Who's On First** gazetteer source to `whosonfirst.org` from `whosonfirst.mapzen.com`.
+  * Refactors to support **RAWR tile** builds in queries/jinja and elsewhere.
+  * Add support for multiple **localized names** from Natural Earth 4.x by adding new transform `convert_ne_l10n_name` for 2-char language codes.
+  * Use a Tilezen curated **country admin polygon** layer to determine country codes PIP for intermediate processing (this layer is not exported in final tiles).
+  * Add new function to **calculate linear overlap with polygons**, useful for road in country calculation.
+  * Add new function to **calculate point in polygon (PIP)**, useful in POI in country calculation.
+  * Updates to Tilezen curated **buffered_land** layer for marine boundary lines.
+  * Be more robust to only add database columns when they don't already exist.
+  * Support easier creation of **generative tests** for points, lines, and polygons.
+  * Continued migration to generative tests instead of live-data OpenStreetMap tests via Overpass.
+  * Switch to generating MVT format with **Coanacatl**, which wraps Wagyu and VTZero.
+  * Requires Postgresql 9.5. [Issue #1319](https://github.com/tilezen/vector-datasource/issues/1319).
+  * Upgrade to **CircleCI 2.0** for continuous integration.
+  * Refactor all OpenStreetMap to Tilezen ETL logic for `network` and `ref` > `shield_text` to support international road shields based on fuzzy data and missing country code data, including specific functions per country.
+  * Add `max_zoom_filter` to remove features with a `max_zoom` if it's < `nominal` zoom.
+  * Add whitelist for **fence** `kind_detail` values.
+  * This will be the last set of **database migrations** provided. All future releases will assume "global" RAWR tile builds.
+
 v1.4.3
 ------
+* **Release date:** 2018-01-08.
+* RAWR internal pre-release round 2.
 * Clip buildings to tile boundaries. See #1142.
 * Allow some kinds of non-numeric `shield_text`. See #1452.
 * Add shops from osm.org and iD. See #1447.
 * Clip buildings to tiles. See #1446.
 * Include all name variants. See #1454.
 * Add building material tag to output. See #1455.
 * Add route modifier information to network. See #1460.
-* Add wetland detail to kind_detail. See #1461.
-* Remove unused wooded area tags + natural=park and add leaf_type to wooded areas. See #1459.
+* Add wetland detail to `kind_detail. See #1461.
```

Comment:
Looks like there might be a missing backtick after `kind_detail`?

Reply:
Fixed in 50f3fca606f12ee49d41643b8994bdca27df561c.